src/git_demo/some_module.py

```diff
@@ -0,0 +1,2 @@
+class Test(object):
+    pass
```

Comment:
Cool module, but I would add this:

```suggestion
    def __init__(self, number):
        self.number = number
```